### PR TITLE
OPERATOR-410 OCI-Monitor pods aren't coming up on OCP (IBM) due to al…

### PR DESCRIPTION
**Pull request by haibinxie**
…l nodes being labels as masters and worker

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by haibinxie** [comment]
@piyush-nimbalkar I have not added/changed unit tests, I tested manually. The main fix is ready to review if you want.

File: drivers/storage/portworx/deployment.go [hunk 1]
```diff
@@ -692,26 +692,68 @@ func (t *template) getSelectorRequirements() []v1.NodeSelectorRequirement {
...
+			Operator: v1.NodeSelectorOpExists,
+		},
+	)
+	requirements2 = append(
+		requirements2,
+		v1.NodeSelectorRequirement{
+			Key:      "node-role.kubernetes.io/worker",
+			Operator: v1.NodeSelectorOpExists,
+		},
+	)
```
Comment (piyush-nimbalkar) [inline drivers/storage/portworx/deployment.go:739]: nit: We can simplify this - 
```
	requirements2 = append(
		requirements2,
		v1.NodeSelectorRequirement{
			Key:      "node-role.kubernetes.io/master",
			Operator: v1.NodeSelectorOpExists,
		},
		v1.NodeSelectorRequirement{
			Key:      "node-role.kubernetes.io/worker",
			Operator: v1.NodeSelectorOpExists,
		},
	)
```

**Review by piyush-nimbalkar — COMMENTED** [review]
This change looks good to me.

**Comment by haibinxie** [comment]
@piyush-nimbalkar I have resolved the comment and added unit tests.

**Review by piyush-nimbalkar — APPROVED** [review]

**Pull request merged**